### PR TITLE
Update copyright headers

### PR DESCRIPTION
**Pull request by marabooy**
### Issues

Updates the copyright headers of all files to that of dotnetframwork.

### Description

*Briefly describe the changes of this pull request.*

### Checklist (Uncheck if it is not completed)

- [ ] *Test cases added*
- [ ] *Build and test with one-click build and test script passed*

### Additional work necessary

*If documentation update is needed, please add "Docs Needed" label to the issue and provide details about the required document change in the issue.*

### CODE REVIEW

File: src/Microsoft.AspNet.OData.Shared/Common/CommonWebApiResources.Designer.cs [hunk 1]
```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="CommonWebApiResources.Designer.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```
Comment (gathogojr) [inline src/Microsoft.AspNet.OData.Shared/Common/CommonWebApiResources.Designer.cs:8]: Does this auto-generated file need to have the copyright header?

File: src/Microsoft.AspNet.OData.Shared/Common/SRResources.Designer.cs [hunk 2]
```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="SRResources.Designer.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```
Comment (gathogojr) [inline src/Microsoft.AspNet.OData.Shared/Common/SRResources.Designer.cs:8]: Does this auto-generated file need the copyright header?

File: src/Microsoft.AspNet.OData.Shared/Query/Expressions/PropertyContainer.generated.cs [hunk 3]
```diff
@@ -1,4 +1,11 @@
-﻿// <auto-generated />
+//-----------------------------------------------------------------------------
+// <copyright file="PropertyContainer.generated.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+// <auto-generated />
```
Comment (gathogojr) [inline src/Microsoft.AspNet.OData.Shared/Query/Expressions/PropertyContainer.generated.cs:8]: Verify if this file is auto-generated. If true then I'd be curious if we need to add the copyright header

File: src/Microsoft.AspNetCore.OData/SingleResultOfT.cs [hunk 4]
```diff
@@ -1,4 +1,11 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+//-----------------------------------------------------------------------------
+// <copyright file="SingleResultOfT.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+// Copyright (c) .NET Foundation. All rights reserved.
```
Comment (gathogojr) [inline src/Microsoft.AspNetCore.OData/SingleResultOfT.cs:8]: Seems this file had a copyright header with .NET Foundation. You may need to remove this line and the one that follows manually

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/DefaultBatchGeneratedClient.cs [hunk 5]
```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="DefaultBatchGeneratedClient.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```
Comment (gathogojr) [inline test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/DefaultBatchGeneratedClient.cs:8]: Verify if this file is auto-generated. If yes, do we need to add the copyright header?

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/UnbufferedBatchGeneratedClient.cs [hunk 6]
```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="UnbufferedBatchGeneratedClient.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```
Comment (gathogojr) [inline test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/UnbufferedBatchGeneratedClient.cs:8]: Verify if this file is auto-generated. If yes, do we need to add the copyright header?

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Common/Controllers/TypeLibrary/NorthwindTypes.cs [hunk 7]
```diff
@@ -1,4 +1,11 @@
-﻿//------------------------------------------------------------------------------
+//-----------------------------------------------------------------------------
+// <copyright file="NorthwindTypes.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
+// </copyright>
+//------------------------------------------------------------------------------
+
+//------------------------------------------------------------------------------
```
Comment (gathogojr) [inline test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Common/Controllers/TypeLibrary/NorthwindTypes.cs:8]: Verify if this file is auto-generated. If yes, which I doubt, do we need to add the copyright header?

File: src/Microsoft.AspNet.OData.Shared/Common/CommonWebApiResources.Designer.cs (hunk 1 above)
Comment (xuzhg) [reply src/Microsoft.AspNet.OData.Shared/Common/CommonWebApiResources.Designer.cs:8]: Need to exclude the auto-generated files out.

**Review by xuzhg — APPROVED** [review]

File: samples/AspNetCore3xEndpointSample.Web/Controllers/CustomersController.cs [hunk 8]
```diff
@@ -1,5 +1,9 @@
-﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
-// Licensed under the MIT License.  See License.txt in the project root for license information.
+//-----------------------------------------------------------------------------
+// <copyright file="CustomersController.cs" company=".NET Foundation">
+//      Copyright (c) .NET Foundation and Contributors. All rights reserved. 
+//      See License.txt in the project root for license information.
```
Comment (xuzhg) [inline samples/AspNetCore3xEndpointSample.Web/Controllers/CustomersController.cs:4]: do we have the correct License.txt in the project root folder?

Comment (marabooy) [reply samples/AspNetCore3xEndpointSample.Web/Controllers/CustomersController.cs:4]: I have updated it in the second pr for the non-sources files.

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Common/Controllers/TypeLibrary/NorthwindTypes.cs (hunk 7 above)
Comment (gathogojr) [reply test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Common/Controllers/TypeLibrary/NorthwindTypes.cs:8]: @marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/UnbufferedBatchGeneratedClient.cs (hunk 6 above)
Comment (gathogojr) [reply test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/UnbufferedBatchGeneratedClient.cs:8]: @marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project

File: test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/DefaultBatchGeneratedClient.cs (hunk 5 above)
Comment (gathogojr) [reply test/E2ETest/Microsoft.Test.E2E.AspNet.OData/Batch/Client/DefaultBatchGeneratedClient.cs:8]: @marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project

File: src/Microsoft.AspNet.OData.Shared/Query/Expressions/PropertyContainer.generated.cs (hunk 3 above)
Comment (gathogojr) [reply src/Microsoft.AspNet.OData.Shared/Query/Expressions/PropertyContainer.generated.cs:8]: @marabooy You can ignore this comment. I think it's right for this file to have the copyright header. It just happens that it was auto-generated somewhere else before being added to this project

**Review by gathogojr — APPROVED** [review]
Left a few comments otherwise LGTM

**Review by wandeg — APPROVED** [review]

**Pull request merged**